### ISSUE TITLE
ExpandableView fill whole page when in Landscape mode. Cannot scroll

### ISSUE THREAD

**Issue by OptivolveThiyraashDavid** [AndreiMisiukevich/ExpandableView#9]
Hi, I have an issue with expandable view in landscape mode. The expandableview works great in portrait mode. But when its in landscape mode, the secondary view just fill up the whole screen with no scroll capability. Its only show show half of secondaryView child. I cannot scroll the secondaryview so another half child view in secondayviewtemplate is out of the screen.?

**Comment by AndreiMisiukevich** [comment]
Hi 
Did you put your views inside scrollview ?

Attach sample please 🙏

What platform btw

**Comment by OptivolveThiyraashDavid** [comment]
Hi sorry i did'nt give detail ,I'm using Xamarin Form(4.0.0.8055-pre1).
This is code and output for the following code after added ScrollView. 
![image](https://user-images.githubusercontent.com/32664570/52698673-f4d62280-2fae-11e9-97b4-643a67b75d77.png)
![ezgif com-resize](https://user-images.githubusercontent.com/32664570/52699868-bb52e680-2fb1-11e9-870f-2a57dc854ede.gif)



As you said I put the expandable view into scroll view I managed to scroll. But the expandable does not open fully. Its just open as per the video. 
I tried,
1. I do set the height of scrollview still I couldn't open it fully.(Same output as above)
2. Set scroll view inside expandable secondary view (Views misplaced)
